app/models/task.rb

```diff
@@ -193,35 +189,11 @@ def reject(reviewer)
     end
   end
 
-  # TODO: Remove value method and use underlying column
-  def award(closer, winning_event, amount=nil)
-    stop_work!(winning_event.user)
-
-    minting = nil
-    award = nil
-    add_activity(closer, Activities::Award) do
-      win = ::Event::Win.new(user: closer)
-      add_event(win) do
-        award = self.awards.create!(
-          awarder: closer,
-          winner: winning_event.user,
-          event: win,
-          wip: self,
-          cents: value
-        )
-
-        minting = mint_coins!(award, amount)
-
-        milestones.each(&:touch)
-      end
-    end
-
-    CoinsMinted.new.perform(minting.id) if minting
-    award
+  def award(closer, winning_event, amount=nil, reason=nil)
+    award_with_reason(closer, winning_event.user, reason, amount)
   end
 
-  # TODO: fix API of above method and combine methods
-  def award_with_reason(closer, winner, reason)
+  def award_with_reason(closer, winner, reason, amount=nil)
```

Comment:
Assignment Branch Condition size for award_with_reason is too high. [18.92/15]<br>Surrounding space missing in default value assignment.